packages/contracts/contracts/core/SphinxModule.sol

```diff
@@ -34,6 +36,42 @@ contract SphinxModule is ReentrancyGuard, Enum, ISphinxModule, Initializable {
      */
     string public constant override VERSION = "1.0.0";
 
+    /**
+     * @dev The code hash for the Gnosis Safe proxy v1.3.0.
+     */
+    bytes32 internal constant SAFE_PROXY_CODE_HASH_1_3_0 =
+        0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000;
```

Comment:
Constants are generally more restrictive than immutables because they need to be defined inline, unlike immutables, which can also be defined in the constructor. Immutables can also read from the blockchain state, whereas constants can't. I think constants can also sometimes be cheaper. I think it's generally considered a best practice to use constants when the variable is known at compile time, and to use immutables for variables that need to be defined in the constructor or read from the chain's state. E.g. [OpenZeppelin's ERC-1967 implementation variable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/proxy/ERC1967/ERC1967Utils.sol#L32-L37) is a constant